util/process_wrapper/process_wrapper.cc

```diff
@@ -79,7 +77,8 @@ int PW_MAIN(int argc, const CharType* argv[], const CharType* envp[]) {
       if (value == PW_SYS_STR("${pwd}")) {
         value = System::GetWorkingDirectory();
       }
-      subst_mappings.push_back({std::move(key), std::move(value)});
+      subst_mappings.push_back({PW_SYS_STR("${") + key + PW_SYS_STR("}"),
```

Comment:
Why is this necessary?

Reply:
We used to do this below and I've opted to just store the mapping as `${key}` rather than `key` since it's never used as `key` anyway.

https://github.com/bazelbuild/rules_rust/blob/82059917e2c7e69244d422478bdb73f8a6d5a7b9/util/process_wrapper/process_wrapper.cc#L176-L180

Reply:
I like it!